meta-resin-common/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -14,6 +15,7 @@ PACKAGECONFIG_append = " systemd modemmanager ppp"
 PACKAGES += "${PN}-bash-completion"
 
 FILES_${PN}-bash-completion = "${datadir}/bash-completion"
+RESIN_BOOT_PARTITION_FILES_append = " system-connections/examples/connection.example:/system-connections/examples/connection.example"
```

Comment:
Also, rename the sample connection to "resin-sample" and also have it directly under the "system-connections"  directory